app/models/requests/JourneyRequest.scala

```diff
@@ -22,6 +22,6 @@ import play.api.mvc.WrappedRequest
 
 class JourneyRequest[A](val authenticatedRequest: AuthenticatedRequest[A], val cacheModel: ExportsDeclaration)
     extends AuthenticatedRequest[A](authenticatedRequest, authenticatedRequest.user) {
-  val choice: Choice = Choice(cacheModel.choice)
+  val choice: Choice = Choice(cacheModel.`type`)
```

Comment:
Arguably this could probably change too but I dont want this PR to grow any further